assets/batch_score/environments/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,21 @@
+name: batch-score-env
+channels:
+  - defaults
+dependencies:
+  - python=3.8
+  - pip
+  - pip:
+      - mlflow
+      - azureml-core
```

Comment:
consider pinning package versions. for reproducibility. we recommend to use a template to resolve the latest on pypi at least for azureml-packages 

=={{latest-pypi-version}}

Reply:
Sounds good. Added.